Bilous/HW6/Help.txt

```diff
@@ -0,0 +1 @@
+I need help!
```

Comment:
how exactly can i help you?

Reply:
You have already helped. I wrote this file, when I  had tried to resolve my problem with merging commit and deleting files. You told me to create and pullrequest the new branch. I have done this. The name of new branch is BilousS. The difference with this branch starts from HW6.